demo/lit-html/base-element.js

```diff
@@ -0,0 +1,28 @@
+import XElement from '../../x-element.js';
```

Comment:
Example of how to get back previous behavior of binding `lit-html` if desired. This is a key test-case to make sure that migration won't require changing _every component at once_ within a codebase.